### PR TITLE
[SUBGRAPH] fix resolver entry

### PR DESCRIPTION
**Pull request by 0xdavinchee**
we move this: `resolverEntry.isToken = Token.load(target.toHex()) != null;` out of the `resolverEntry == null` and only update it if the `target` is not null.

We need to do this because of sepolia where the token entity is created after the resolver set event is emitted (triggered), thus we want to be able to call set again and fix this. Edited: This makes ETHx not shown as listed in sepolia subgraph.

### CODE REVIEW

File: packages/subgraph/src/mappings/resolver.ts [hunk 1]
```diff
@@ -47,14 +42,34 @@ export function handleRoleRevoked(event: RoleRevoked): void {
...
+        resolverEntry.createdAtTimestamp = block.timestamp;
+        resolverEntry.targetAddress = target;
+    }
+    const isListed = target.notEqual(ZERO_ADDRESS);
+
+    // we only update this if the target is not equal to the zero address
+    if (isListed) {
+        resolverEntry.isToken = Token.load(target.toHex()) != null;
+    }
+    resolverEntry.updatedAtBlockNumber = block.number;
```
Comment (kasparkallas) [inline packages/subgraph/src/mappings/resolver.ts:60]: is it ok to always update when `getOrInit` is called? it's called in very few places?

**Review by kasparkallas — APPROVED** [review]
acknowledged

**Comment by hellwolf** [comment]
how is this one doing

**Comment by hellwolf** [comment]
the fix did not work, will be investigated.

**Comment by hellwolf** [comment]
@elvijsTDL please write a concrete bug report on the issue related to this.

**Comment by elvijsTDL** [comment]
The subgraph shows some of the tokens as unlisted even if they should be e.g. ETHx , fUSDCx

If you query the listed tokens you can see that they are missing:

``` query MyQuery {
  tokens(where: {isListed: true}) {
    isListed
    isSuperToken
    isNativeAssetSuperToken
    name
    symbol
  }
}
```

Screenshot 2024-03-22 at 11 56 16

At some point it has flipped from listed to unlisted as it used to work fine 

This might be related to GoodDollar G$ getting unlisted from Celo too

**Pull request merged**